resources/views/tabbar/navigation/contents/it/event/connpass_list.blade.php

```diff
@@ -1,6 +1,6 @@
-<span ng-repeat="count in $ctrl.getCountArray($ctrl.getJson['results_available'])" style="display:inline;">
-    <a href="/contents/?contents=connpass&year=@{{$ctrl.yearParam}}&month=@{{$ctrl.monthParam}}&keyword=@{{$ctrl.keywordParam}}&page=@{{(count + 1) / $ctrl.countParam}}" ng-if="(count + 1) % $ctrl.countParam == 0">@{{(count + 1) / $ctrl.countParam}} </a>
-    <a href="/contents/?contents=connpass&year=@{{$ctrl.yearParam}}&month=@{{$ctrl.monthParam}}&keyword=@{{$ctrl.keywordParam}}&page=@{{$ctrl.Math.ceil((count + 1) / $ctrl.countParam)}}" ng-if="count + 1 == $ctrl.getJson['results_available'] && (count + 1) % $ctrl.countParam != 0">@{{$ctrl.Math.ceil((count + 1) / $ctrl.countParam)}}</a>
+<span ng-repeat="count in $ctrl.connpassService.getCountArray($ctrl.getJson['results_available'])" style="display:inline;">
```

Comment:
__[code_sniffer]__
* No PHP code was found in this file and short open tags are not allowed by this install of PHP. This file may be using short open tags but PHP does not allow them.